seeds/value_sets/cms_hcc/cms_hcc_seeds.yml

```diff
@@ -25,8 +25,12 @@ seeds:
       tags: cms_hcc
       enabled: "{{ var('cms_hcc_enabled',var('tuva_marts_enabled',True)) }}"
       column_types:
+        payment_year: |
+          {%- if target.type in ("bigquery", "databricks") -%} string {%- else -%} varchar {%- endif -%}
```

Comment:
For Databricks, it's not the data type itself that matters - it more matters that a type was specified because type inference for large seeds was found to be excruciatingly slow!

Changed to `integer` and all works fine, see my latest commit.